src/pykeen/models/base.py

```diff
@@ -569,41 +584,27 @@ def predict_scores_all_heads(
         # Enforce evaluation mode
         self.eval()
 
-        '''
-        In case the model was trained using inverse triples, the scoring of all heads is not handled by calculating
-        the scores for all heads based on a (relation, tail) pair, but instead all possible tails are calculated
-        for a (tail, inverse_relation) pair.
-        '''
         if not self.triples_factory.create_inverse_triples:
+            '''
+            In case the model was trained using inverse triples, the scoring of all heads is not handled by calculating
+            the scores for all heads based on a (relation, tail) pair, but instead all possible tails are calculated
+            for a (tail, inverse_relation) pair.
+            '''
             if slice_size is None:
                 scores = self.score_h(rt_batch)
             else:
                 scores = self.score_h(rt_batch, slice_size=slice_size)
-            if self.predict_with_sigmoid:
-                scores = torch.sigmoid(scores)
-            return scores
-
-        '''
-        The PyKEEN package handles _inverse relations_ by adding the number of relations to the indices of the
-        _native relation_.
-        Example:
-        The triples/knowledge graph used to train the model contained 100 relations. Due to using inverse relations,
-        the model now has an additional 100 inverse relations. If the _native relation_ has the index 3, the index
-        of the _inverse relation_ is 4 (id of relation + 1).
-        '''
-        rt_batch_cloned = rt_batch.clone()
-        rt_batch_cloned.to(device=rt_batch.device)
-
-        # The number of relations stored in the triples factory includes the number of inverse relations
-        # Id of inverse relation: relation + 1
-        rt_batch_cloned[:, 0] = rt_batch_cloned[:, 0] + 1
-
-        # The score_t function requires (entity, relation) pairs instead of (relation, entity) pairs
-        rt_batch_cloned = rt_batch_cloned.flip(1)
-        if slice_size is None:
-            scores = self.score_t(rt_batch_cloned)
         else:
-            scores = self.score_t(rt_batch_cloned, slice_size=slice_size)
+            '''
+            The PyKEEN package handles _inverse relations_ by adding the number of relations to the indices of the
```

Comment:
the comment is wrong.
Also it is not needed here.

Reply:
so just delete it?

Reply:
Done in 324d5b3e